pkg/ovssubnet/controller/kube/bin/openshift-sdn-kube-subnet-setup.sh

```diff
@@ -147,11 +141,15 @@ function setup() {
     echo "export OPENSHIFT_CLUSTER_SUBNET=${cluster_network_cidr}" >> "/etc/openshift-sdn/config.env"
 
     # delete unnecessary routes
-    delete_local_subnet_route lbr0
+    delete_local_subnet_route lbr0 || true
```

Comment:
@pravisankar I'm not sure why, but when running the SDN from within a container this route never shows up so I've added this to tolerate it's non-existence.